modules/end-user-guide/partials/proc_adding-vs-code-extensions.adoc

```diff
@@ -0,0 +1,58 @@
+
+[id="adding-vs-code-extensions"]
+= Adding `VS Code` extensions
```

Comment:
```suggestion
= Automating VS Code extensions installation at worksapce startup
```

Adding an extension is about manually installing a VS Code extensions via the extensions view. But this section is about something else: it's about configuring your repository so that the extensions get installed automatically.

Reply:
Wonderful! Thank you. I'm applying this suggestion.